src/test/java/formflow/library/file/UploadJourneyTests.java

```diff
@@ -58,6 +61,7 @@ void documentUploadFlow() {
     } catch (InterruptedException e) {
       throw new RuntimeException(e);
     }
+    takeSnapShot("test.png");
```

Comment:
Do we need this here beyond this ticket?   (I realize this isn't ready for review yet, just wondering)

Reply:
Nope, accidentally left that in.